llvm/lib/Target/AMDGPU/SIInstructions.td

```diff
@@ -2192,20 +2192,6 @@ foreach pred = [NotHasTrue16BitInsts, UseFakeTrue16Insts] in {
   }
 }
 
-let True16Predicate = UseRealTrue16Insts in {
```

Comment:
Why are you removing these patterns?

Reply:
I just realized these are patterns merged last week. Restored